[2024-05-22 02:20:40,254] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T21:00:00+00:00 [queued]>
[2024-05-22 02:20:40,569] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T21:00:00+00:00 [queued]>
[2024-05-22 02:20:40,576] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 02:20:40,584] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2024-05-22 02:20:40,599] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 02:20:40,873] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): fetch_data> on 2024-05-21 21:00:00+00:00
[2024-05-22 02:20:40,998] {standard_task_runner.py:52} INFO - Started process 2406 to run task
[2024-05-22 02:20:41,073] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'purpleair_api_dag', 'fetch_data', 'scheduled__2024-05-21T21:00:00+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion.py', '--cfg-path', '/tmp/tmpxxzk9khu', '--error-file', '/tmp/tmppcffpp9f']
[2024-05-22 02:20:41,233] {standard_task_runner.py:77} INFO - Job 23: Subtask fetch_data
[2024-05-22 02:20:42,251] {logging_mixin.py:109} INFO - Running <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T21:00:00+00:00 [running]> on host 8fe6997c1526
[2024-05-22 02:20:43,012] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=purpleair_api_dag
AIRFLOW_CTX_TASK_ID=fetch_data
AIRFLOW_CTX_EXECUTION_DATE=2024-05-21T21:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-21T21:00:00+00:00
[2024-05-22 02:20:43,245] {logging_mixin.py:109} INFO - Failed to retrieve data: 403
[2024-05-22 02:20:43,249] {python.py:175} INFO - Done. Returned value was: None
[2024-05-22 02:20:43,321] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=purpleair_api_dag, task_id=fetch_data, execution_date=20240521T210000, start_date=20240522T022040, end_date=20240522T022043
[2024-05-22 02:20:43,447] {local_task_job.py:154} INFO - Task exited with return code 0
[2024-05-22 02:20:43,834] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-22 04:17:55,088] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T21:00:00+00:00 [queued]>
[2024-05-22 04:17:55,179] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T21:00:00+00:00 [queued]>
[2024-05-22 04:17:55,198] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 04:17:55,199] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2024-05-22 04:17:55,200] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 04:17:55,293] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): fetch_data> on 2024-05-21 21:00:00+00:00
[2024-05-22 04:17:55,337] {standard_task_runner.py:52} INFO - Started process 386 to run task
[2024-05-22 04:17:55,380] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'purpleair_api_dag', 'fetch_data', 'scheduled__2024-05-21T21:00:00+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion.py', '--cfg-path', '/tmp/tmpxn_9kj2e', '--error-file', '/tmp/tmp1lpcx4ac']
[2024-05-22 04:17:55,423] {standard_task_runner.py:77} INFO - Job 41: Subtask fetch_data
[2024-05-22 04:17:55,707] {logging_mixin.py:109} INFO - Running <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T21:00:00+00:00 [running]> on host 3419eee920a5
[2024-05-22 04:17:55,999] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=purpleair_api_dag
AIRFLOW_CTX_TASK_ID=fetch_data
AIRFLOW_CTX_EXECUTION_DATE=2024-05-21T21:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-21T21:00:00+00:00
[2024-05-22 04:17:56,012] {logging_mixin.py:109} INFO -  api is None
[2024-05-22 04:17:56,030] {logging_mixin.py:109} INFO - Full URL: https://api.purpleair.com/v1/sensors?fields=latitude%2Clongitude%2Chumidity%2Cpm2.5_60minute%2Clast_seen&location_type=0&max_age=300
[2024-05-22 04:17:56,041] {logging_mixin.py:109} INFO - Headers: {'X-API-Key': 'None'}
[2024-05-22 04:17:56,157] {logging_mixin.py:109} INFO - Status Code: 403
[2024-05-22 04:17:56,158] {logging_mixin.py:109} INFO - Response Content: {
  "api_version" : "V1.0.13-0.0.56",
  "time_stamp" : 1716351476,
  "error" : "ApiKeyInvalidError",
  "description" : "The provided api_key was not valid."
}
[2024-05-22 04:17:56,257] {logging_mixin.py:109} INFO - Failed to retrieve data: 403
[2024-05-22 04:17:56,258] {python.py:175} INFO - Done. Returned value was: None
[2024-05-22 04:17:56,297] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=purpleair_api_dag, task_id=fetch_data, execution_date=20240521T210000, start_date=20240522T041755, end_date=20240522T041756
[2024-05-22 04:17:56,394] {local_task_job.py:154} INFO - Task exited with return code 0
[2024-05-22 04:17:56,467] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-22 06:56:32,994] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T21:00:00+00:00 [queued]>
[2024-05-22 06:56:33,059] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T21:00:00+00:00 [queued]>
[2024-05-22 06:56:33,059] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 06:56:33,060] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2024-05-22 06:56:33,067] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 06:56:33,217] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): fetch_data> on 2024-05-21 21:00:00+00:00
[2024-05-22 06:56:33,252] {standard_task_runner.py:52} INFO - Started process 572 to run task
[2024-05-22 06:56:33,338] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'purpleair_api_dag', 'fetch_data', 'scheduled__2024-05-21T21:00:00+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion.py', '--cfg-path', '/tmp/tmpxnz8mzw0', '--error-file', '/tmp/tmpa3z0_fhf']
[2024-05-22 06:56:33,416] {standard_task_runner.py:77} INFO - Job 57: Subtask fetch_data
[2024-05-22 06:56:33,949] {logging_mixin.py:109} INFO - Running <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T21:00:00+00:00 [running]> on host 148b4063f71c
[2024-05-22 06:56:34,371] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=purpleair_api_dag
AIRFLOW_CTX_TASK_ID=fetch_data
AIRFLOW_CTX_EXECUTION_DATE=2024-05-21T21:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-21T21:00:00+00:00
[2024-05-22 06:56:34,480] {logging_mixin.py:109} INFO - Full URL: https://api.purpleair.com/v1/sensors?fields=latitude%2Clongitude%2Chumidity%2Cpm2.5_60minute%2Ctemperature&location_type=0&max_age=300
[2024-05-22 06:56:34,489] {logging_mixin.py:109} INFO - Headers: {'X-API-Key': '9A23CD90-1279-11EF-B9F7-42010A80000D'}
[2024-05-22 06:56:34,797] {logging_mixin.py:109} INFO - Status Code: 402
[2024-05-22 06:56:34,798] {logging_mixin.py:109} INFO - Response Content: {
  "api_version" : "V1.0.13-0.0.56",
  "time_stamp" : 1716360994,
  "error" : "PaymentRequiredError",
  "description" : "Payment is required to make this api call."
}
[2024-05-22 06:56:35,012] {logging_mixin.py:109} INFO - Failed to retrieve data: 402
[2024-05-22 06:56:35,022] {python.py:175} INFO - Done. Returned value was: None
[2024-05-22 06:56:35,415] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=purpleair_api_dag, task_id=fetch_data, execution_date=20240521T210000, start_date=20240522T065633, end_date=20240522T065635
[2024-05-22 06:56:35,601] {local_task_job.py:154} INFO - Task exited with return code 0
[2024-05-22 06:56:35,932] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-22 07:04:55,597] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T21:00:00+00:00 [queued]>
[2024-05-22 07:04:55,705] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T21:00:00+00:00 [queued]>
[2024-05-22 07:04:55,720] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 07:04:55,721] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2024-05-22 07:04:55,722] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 07:04:55,833] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): fetch_data> on 2024-05-21 21:00:00+00:00
[2024-05-22 07:04:55,884] {standard_task_runner.py:52} INFO - Started process 1121 to run task
[2024-05-22 07:04:55,915] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'purpleair_api_dag', 'fetch_data', 'scheduled__2024-05-21T21:00:00+00:00', '--job-id', '137', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion.py', '--cfg-path', '/tmp/tmp7icjl9j5', '--error-file', '/tmp/tmpa1e2zuz3']
[2024-05-22 07:04:55,980] {standard_task_runner.py:77} INFO - Job 137: Subtask fetch_data
[2024-05-22 07:04:56,351] {logging_mixin.py:109} INFO - Running <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T21:00:00+00:00 [running]> on host 148b4063f71c
[2024-05-22 07:04:56,537] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=purpleair_api_dag
AIRFLOW_CTX_TASK_ID=fetch_data
AIRFLOW_CTX_EXECUTION_DATE=2024-05-21T21:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-21T21:00:00+00:00
[2024-05-22 07:04:56,546] {logging_mixin.py:109} INFO - Full URL: https://api.purpleair.com/v1/sensors?fields=latitude%2Clongitude%2Chumidity%2Cpm2.5_60minute%2Ctemperature&location_type=0&max_age=300
[2024-05-22 07:04:56,552] {logging_mixin.py:109} INFO - Headers: {'X-API-Key': '9A23CD90-1279-11EF-B9F7-42010A80000D'}
[2024-05-22 07:04:56,795] {logging_mixin.py:109} INFO - Status Code: 402
[2024-05-22 07:04:56,819] {logging_mixin.py:109} INFO - Response Content: {
  "api_version" : "V1.0.13-0.0.56",
  "time_stamp" : 1716361496,
  "error" : "PaymentRequiredError",
  "description" : "Payment is required to make this api call."
}
[2024-05-22 07:04:57,052] {logging_mixin.py:109} INFO - Failed to retrieve data: 402
[2024-05-22 07:04:57,053] {python.py:175} INFO - Done. Returned value was: None
[2024-05-22 07:04:57,160] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=purpleair_api_dag, task_id=fetch_data, execution_date=20240521T210000, start_date=20240522T070455, end_date=20240522T070457
[2024-05-22 07:04:57,407] {local_task_job.py:154} INFO - Task exited with return code 0
[2024-05-22 07:04:58,238] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-22 07:12:44,926] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T21:00:00+00:00 [queued]>
[2024-05-22 07:12:45,071] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T21:00:00+00:00 [queued]>
[2024-05-22 07:12:45,077] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 07:12:45,077] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2024-05-22 07:12:45,079] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 07:12:45,180] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): fetch_data> on 2024-05-21 21:00:00+00:00
[2024-05-22 07:12:45,225] {standard_task_runner.py:52} INFO - Started process 1656 to run task
[2024-05-22 07:12:45,244] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'purpleair_api_dag', 'fetch_data', 'scheduled__2024-05-21T21:00:00+00:00', '--job-id', '217', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion.py', '--cfg-path', '/tmp/tmpe17tw1vk', '--error-file', '/tmp/tmp2rroho3y']
[2024-05-22 07:12:45,316] {standard_task_runner.py:77} INFO - Job 217: Subtask fetch_data
[2024-05-22 07:12:45,710] {logging_mixin.py:109} INFO - Running <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T21:00:00+00:00 [running]> on host 148b4063f71c
[2024-05-22 07:12:46,093] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=purpleair_api_dag
AIRFLOW_CTX_TASK_ID=fetch_data
AIRFLOW_CTX_EXECUTION_DATE=2024-05-21T21:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-21T21:00:00+00:00
[2024-05-22 07:12:46,145] {logging_mixin.py:109} INFO - Full URL: https://api.purpleair.com/v1/sensors?fields=latitude%2Clongitude%2Chumidity%2Cpm2.5_60minute%2Ctemperature&location_type=0&max_age=300
[2024-05-22 07:12:46,146] {logging_mixin.py:109} INFO - Headers: {'X-API-Key': '9A23CD90-1279-11EF-B9F7-42010A80000D'}
[2024-05-22 07:12:46,686] {logging_mixin.py:109} INFO - Status Code: 402
[2024-05-22 07:12:46,687] {logging_mixin.py:109} INFO - Response Content: {
  "api_version" : "V1.0.13-0.0.56",
  "time_stamp" : 1716361966,
  "error" : "PaymentRequiredError",
  "description" : "Payment is required to make this api call."
}
[2024-05-22 07:12:46,921] {logging_mixin.py:109} INFO - Failed to retrieve data: 402
[2024-05-22 07:12:46,979] {python.py:175} INFO - Done. Returned value was: None
[2024-05-22 07:12:47,518] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=purpleair_api_dag, task_id=fetch_data, execution_date=20240521T210000, start_date=20240522T071244, end_date=20240522T071247
[2024-05-22 07:12:47,926] {local_task_job.py:154} INFO - Task exited with return code 0
[2024-05-22 07:12:48,154] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-22 07:22:27,491] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T21:00:00+00:00 [queued]>
[2024-05-22 07:22:27,634] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T21:00:00+00:00 [queued]>
[2024-05-22 07:22:27,645] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 07:22:27,649] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2024-05-22 07:22:27,650] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 07:22:27,836] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): fetch_data> on 2024-05-21 21:00:00+00:00
[2024-05-22 07:22:27,888] {standard_task_runner.py:52} INFO - Started process 2351 to run task
[2024-05-22 07:22:27,943] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'purpleair_api_dag', 'fetch_data', 'scheduled__2024-05-21T21:00:00+00:00', '--job-id', '326', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion.py', '--cfg-path', '/tmp/tmp7ew58tva', '--error-file', '/tmp/tmpq0ra2rff']
[2024-05-22 07:22:28,056] {standard_task_runner.py:77} INFO - Job 326: Subtask fetch_data
[2024-05-22 07:22:28,923] {logging_mixin.py:109} INFO - Running <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-21T21:00:00+00:00 [running]> on host 148b4063f71c
[2024-05-22 07:22:29,537] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=purpleair_api_dag
AIRFLOW_CTX_TASK_ID=fetch_data
AIRFLOW_CTX_EXECUTION_DATE=2024-05-21T21:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-21T21:00:00+00:00
[2024-05-22 07:22:29,602] {logging_mixin.py:109} INFO - Full URL: https://api.purpleair.com/v1/sensors?fields=latitude%2Clongitude%2Chumidity%2Cpm2.5_60minute%2Ctemperature&location_type=0&max_age=300
[2024-05-22 07:22:29,602] {logging_mixin.py:109} INFO - Headers: {'X-API-Key': '9A23CD90-1279-11EF-B9F7-42010A80000D'}
[2024-05-22 07:22:30,103] {logging_mixin.py:109} INFO - Status Code: 402
[2024-05-22 07:22:30,120] {logging_mixin.py:109} INFO - Response Content: {
  "api_version" : "V1.0.13-0.0.56",
  "time_stamp" : 1716362550,
  "error" : "PaymentRequiredError",
  "description" : "Payment is required to make this api call."
}
[2024-05-22 07:22:30,524] {logging_mixin.py:109} INFO - Failed to retrieve data: 402
[2024-05-22 07:22:30,535] {python.py:175} INFO - Done. Returned value was: None
[2024-05-22 07:22:30,974] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=purpleair_api_dag, task_id=fetch_data, execution_date=20240521T210000, start_date=20240522T072227, end_date=20240522T072230
[2024-05-22 07:22:31,217] {local_task_job.py:154} INFO - Task exited with return code 0
[2024-05-22 07:22:31,632] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
